### PR TITLE
feat(TNLT-4216): Adds supplement lead 1 and 1 slice

### PR DESCRIPTION
**Pull request by c-j-j**
Designs: https://app.zeplin.io/project/5eec2dafc5ee3d4cc931fd86/dashboard?q=lifestyle

**iPad Pro 12.9 Landscape**
Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2020-09-02 at 09 58 52

**iPad Pro 12.9 Portrait**
Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2020-09-02 at 09 58 44

**ipad Mini 4 Landscape**
Simulator Screen Shot - iPad mini 4 - 2020-09-02 at 09 42 28
**ipad Mini 4 Portrait**
Simulator Screen Shot - iPad mini 4 - 2020-09-02 at 09 42 19

**iPad 7th gen landscape**
Simulator Screen Shot - iPad (7th generation) - 2020-09-02 at 09 29 46
**iPad 7th gen portrait**
Simulator Screen Shot - iPad (7th generation) - 2020-09-02 at 09 29 40

**Android Pixel C**
image

### CODE REVIEW

File: packages/edition-slices/src/slices/supplementleadoneandone/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { SupplementLeadOneAndOneSlice } from "@times-components-native/slice-layout";
+import { TileU, TileAX } from "../../tiles";
+import { ResponsiveSlice } from "../shared";
+
+import styleFactory from "./styles";
+
+class SupplementLeadOneAndOne extends Component {
```
Comment (Geoff-Ford) [inline packages/edition-slices/src/slices/supplementleadoneandone/index.js:9]: 🤔

File: packages/edition-slices/src/tiles/tile-ax/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  tile,
+  orientation,
+  breakpoint = editionBreakpoints.medium,
+}) => {
+  const styles = stylesFactory(breakpoint);
+  const crop = getTileImage(tile, "crop32");
+
+  if (!crop) {
+    return null;
+  }
```
Comment (Geoff-Ford) [inline packages/edition-slices/src/tiles/tile-ax/index.js:26]: Now I know you are winding me up! Incidentally, I missed one on my last PR 😱  . Such is copy/paste as a starting point from the original tile.

File: packages/slice-layout/src/templates/supplementleadoneandone/styles.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+const wideBreakpointStyles = {
+  ...styles,
+  container: {
+    ...styles.container,
+    marginHorizontal: spacing(2),
+  },
+};
+
+const stylesToreturn = {
```
Comment (Geoff-Ford) [inline packages/slice-layout/src/templates/supplementleadoneandone/styles.js:32]: `stylesToReturn`?

**Review by Geoff-Ford — APPROVED** [review]

File: packages/edition-slices/src/tiles/tile-ax/index.js (hunk 2 above)
Comment (c-j-j) [reply packages/edition-slices/src/tiles/tile-ax/index.js:26]: I think most of the tiles have this

File: packages/edition-slices/src/slices/supplementleadoneandone/index.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          />
+        }
+      />
+    );
+  };
+
+  return <ResponsiveSlice renderSmall={() => null} renderMedium={render} />;
+};
+
+SupplementLeadOneAndOne.propTypes = {
```
Comment (aronshamash) [inline packages/edition-slices/src/slices/supplementleadoneandone/index.js:43]: I have been not including propTypes for new files. What do we think ?

**Review by aronshamash — APPROVED** [review]

**Pull request merged**